type-definitions/immutable.js.flow

```diff
@@ -32,6 +32,21 @@
 // some constructors and functions.
 type PlainObjInput<K, V> = {[key: K]: V, __proto__: null};
 
+// Helper types to extract the "keys" and "values" use by the *In() methods.
+// Exported for experimental use only, may change.
+export type $KeyOf<C> = $Call<
+  & (<K>(?_Collection<K, *>) => K)
```

Comment:
Instead of `*`, I recommend using `_Collection<K, mixed>` here 😊

Reply:
Tried using `mixed` at first, but get errors about value types that don't flow to mixed. I may try `any` instead